docs/reference/search/search-your-data/learning-to-rank-search-usage.asciidoc

```diff
@@ -0,0 +1,77 @@
+[[learning-to-rank-search-usage]]
+=== Search using Learning To Rank
+++++
+<titleabbrev>Search using Learning To Rank</titleabbrev>
+++++
+
+preview::["The Learning To Rank feature is in technical preview and may be changed or removed in a future release. Elastic will work to fix any issues, but this feature is not subject to the support SLA of official GA features."]
+
+NOTE: This feature is available for Elastic Stack versions 8.12.0 and newer and requires a https://www.elastic.co/pricing[Platinum subscription] or higher.
+
+[discrete]
+[[learning-to-rank-rescorer]]
+==== Learning To Rank as a rescorer
+
+Once your LTR model is trained and deployed into {es}, it is possible to use it as a <<rescore, rescorer>> in the Elasticsearch <<search-your-data, Search API>>:
+
+[source,console]
+----
+GET my-index/_search
+{
+  "query": { <1>
+    "multi_match": {
+      "fields": ["title", "content"],
+      "query": "the quick brown fox"
+    }
+  },
+  "rescore": {
+    "learning_to_rank": {
+      "model_id": "ltr-model", <2>
+      "params": { <3>
+        "query_text": "the quick brown fox"
+      }
+    },
+    "window_size": 100 <4>
+  }
+}
+----
+// TEST[skip:TBD]
+<1> First pass query providing documents to be rescored.
+<2> The unique identifier of the trained model uploaded to {es}.
+<3> Named parameters to be passed to the query templates used for feature.
+<4> The number of documents that should be examined by the rescorer on each shard.
+
+[discrete]
+[[learning-to-rank-rescorer-limitations]]
+===== Known limitations
+
+[discrete]
+[[learning-to-rank-rescorer-limitations-window-size]]
+====== Rescore window size
+
+Scores returned by LTR models are usually not comparable with the scores issued by the first pass query and can be lower than the non-rescored score. This can cause the non-rescored result document to be ranked higher than the rescored document.  To prevent this, the window_size parameter is mandatory for LTR rescorers and should be greater than or equal to `from + size`.
```

Comment:
```suggestion
Scores returned by LTR models are usually not comparable with the scores issued by the first pass query and can be lower than the non-rescored score. This can cause the non-rescored result document to be ranked higher than the rescored document. To prevent this, the `window_size` parameter is mandatory for LTR rescorers and should be greater than or equal to `from + size`.
```